README.md

```diff
@@ -1,13 +1,32 @@
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```

Comment:
with `https://github.com/Corniel/dotnet-project-files-analyzers/blob/main/license.md` you get the actual license including the copyright statement.